app/webpacker/css/admin/connected_apps.scss

```diff
@@ -0,0 +1,15 @@
+.connected-app__head {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 1em;
+
+  h3 {
+    margin-bottom: 1em;
```

Comment:
I would like to think we already have reasonable margins for h3 and p by default, but I guess not.

Reply:
I would have thought so, too. But they must be overridden somewhere. Maybe it will be better in the new design?